src/components/pages/enterprise/training/training.jsx

```diff
@@ -34,6 +36,15 @@ const items = [
     buttonLink: 'https://isovalent.com/labs/',
     buttonTarget: '_blank',
   },
+  {
+    logoName: 'soloio',
+    title: 'Introduction to Cilium with Certification Option',
+    description:
+      'Cilium is an open source software for providing, securing and observing network connectivity between container workloads - cloud native, and fueled by the revolutionary Kernel technology eBPF. In this workshop, you will learn the essential skills to deploy the Cilium CNI on a test Kubernetes cluster, gather metrics and enforce network policies. At the completion of the workshop, you will be able to take an assessment and with a score 80% or higher you will earn the “Fundamentals for Cilium” credential, offered by Solo.io with Credly.',
```

Comment:
The first sentence explains what Cilium is, without any relation to the workshop. There is no need to do that at this position, I'd drop the first sentence.

Reply:
The first sentence explains what Cilium is, it's in every workshop that we deliver and I'm looking to maintain consistency. I am okay with leaving it.

Reply:
@distributethe6ix I think @liquidat's comment is aimed more about maintaining consistency with the rest of the Cilium site